CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Recent and upcoming changes to lightdash
  - Stick sidebar on top left 
  - The stuck problem with scrollbar that shows only limited content on overflow
  - Errors in `meta` tags (such as invalid references to other dimensions) no longer crash the server and errors are shown in the UI
+ - UI fixes - Error in input focus
```

Comment:
Again, contribution style, this should be a very user friendly message (not for developers).

Perhaps: "fix in the UI where the search box would be cutoff"